students/Michael Hrstka/Lab 2/sensor/sensor.ino

```diff
@@ -0,0 +1,59 @@
+const int apin=0;
+int val=0;
+float voltage;
+
+void setup() {
+  Serial.begin(9600);
+  analogReference(EXTERNAL); // Use 3.3V connected to AREF
+  pinMode(2, OUTPUT);
+  pinMode(3, OUTPUT);
+  pinMode(4, OUTPUT);
+  pinMode(5, OUTPUT);
+  pinMode(6, OUTPUT);
+  pinMode(7, OUTPUT);
+  pinMode(8, OUTPUT);
+  pinMode(9, OUTPUT);
+}
+
+void setLights(int a, int b, int c, int d, int e, int f, int g, int h){
```

Comment:
Oh man, this is just begging to be handled with a binary register.
But even in Arduino, you could probably simplify the code using an array or something. In general, you want to avoid blocks of (nearly) repeated commands if you can. It makes editing/troubleshooting easier.